src/applications/personalization/profile/components/personal-health-care-contacts/Contacts.jsx

```diff
@@ -86,27 +67,18 @@ const Contacts = ({ data }) => {
         </va-additional-info>
       </div>
 
-      <section className="profile-info-card">
-        <div className="row vads-u-border-color--gray-lighter vads-u-color-gray-dark vads-u-display--flex vads-u-flex-direction--column vads-u-padding-x--2 vads-u-padding-y--1p5 medium-screen:vads-u-padding--4 vads-u-border--1px">
-          <h2 className="vads-u-font-family--sans vads-u-font-size--base vads-u-margin--0">
-            Medical emergency contacts
-          </h2>
-          <p className="vads-u-color--gray-medium vads-u-margin-top--1 vads-u-margin-bottom--1">
-            The people we’ll contact in an emergency.
-          </p>
-          {renderEmergencyContacts}
-        </div>
+      <ProfileInfoCard
+        title="Emergency contacts"
+        data={renderEmergencyContacts}
+        namedAnchor="emergency-contacts"
+      />
```

Comment:
I was looking at the heading map, and I _think_ the title for each card should be set to an h2 and then the title of each contact can stay as an h3. The `ProfileInfoCard` has a level prop that should suffice for this.

Right now the heading map looks like this:
![Screenshot 2024-03-05 at 9 13 37 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/8332986/c92c5420-fec7-49b8-853e-554835db6cd3)

vs the other pages will usually look more like this:
![Screenshot 2024-03-05 at 9 13 54 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/8332986/97d3701e-d189-4b32-aa69-005284cc1928)

side not: we do recognize the issue with the profile menu heading, and there has been talk about fixing that heading mismatch, but it is widespread throughout all the va pages with a sidebar, and hasn't been solved yet, but the design system team is working on that with content/IA folks.